docs/source/nwb_extension_testing.rst

```diff
@@ -0,0 +1,50 @@
+NWB Extension Testing Workflow
+==============================
```

Comment:
@rly I don't think we are currently documenting GitHub workflows in the docs. It's fine if you want to keep it, I'm just wondering whether this makes more sense as documentation in the yaml file